python/ray/tests/test_advanced_2.py

```diff
@@ -685,36 +684,6 @@ def get_neuron_core_ids(neuron_cores_per_worker):
     nc_f = ray.remote(resources={"neuron_cores": 2})(lambda: get_neuron_core_ids(2))
     assert ray.get(nc_f.remote()) == 2
 
-    with pytest.raises(ValueError):
```

Comment:
why are we removing this?

Reply:
Because we now allow specifying both gpu and neuro core resources for a single task.